composer.json

```diff
@@ -40,7 +40,7 @@
         "symfony/twig-bridge": "^2.3.5 || ^3.0",
         "symfony/validator": "^2.3 || ^3.0",
         "twig/extensions": "^1.0",
-        "twig/twig": "^1.26"
+        "twig/twig": "^1.28"
```

Comment:
Yeah but I think we deemed it acceptable and already bumped that in several PRs.